File: java/hcx-integrator-sdk/src/test/java/io/hcxprotocol/dto/ResponseErrorTest.java
package io.hcxprotocol.dto;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;


class ResponseErrorTest {


    @Test
    void ResponseErrorTest() {

        ResponseError responseError = new ResponseError("code", "message", "trace");

        responseError.setCode("code");
        responseError.setMessage("message");
        responseError.setTrace("trace");

        assertEquals("code", responseError.getCode());
        assertEquals("message", responseError.getMessage());
        assertEquals("trace", responseError.getTrace());

    }
}

File: dot-net/hcx-integrator-sdk/UnitTest/Dto/ResponseErrorTest.cs
using Io.HcxProtocol.Dto;
using Microsoft.VisualStudio.TestTools.UnitTesting;

namespace UnitTest.Dto
{
    [TestClass]
    public class ResponseErrorTest
    {
        [TestMethod]
        public void ResponseErrorUnitTest()
        { 
            ResponseError responseError = new ResponseError("code", "message", "trace");
          
            Assert.AreEqual("code", responseError.Code);
            Assert.AreEqual("message", responseError.Message);
            Assert.AreEqual("trace", responseError.Trace);
        }
    }
}
